apps/testnorge-aareg/Dockerfile

```diff
@@ -3,6 +3,6 @@ LABEL maintainer="Team Dolly"
 
 ADD apps/testnorge-aareg/build/libs/app.jar /app/app.jar
 
-ENV JAVA_OPTS="-Dspring.profiles.active=prod"
+ENV JAVA_OPTS="--add-opens java.base/java.lang=ALL-UNNAMED"
```

Comment:
Mulig dette virker for testnorge-aareg, men jeg endte faktisk opp med dette for ident-pool:

I Dockerfile:
ENV JAVA_OPTS="-Dspring.profiles.active=prod --add-opens java.base/java.lang=ALL-UNNAMED"

og fjernet tilsvarende fra nais.yaml